tests/test_package_generation.py

```diff
@@ -0,0 +1,11 @@
+def test_bake_project(cookies):
+    result = cookies.bake(extra_context={"repository_name": "my_project"})
+
+    assert result.exit_code == 0
+    assert result.exception is None
+
+    assert result.project_path.name == "my_project"
+    assert result.project_path.is_dir()
+
+    assert result.project.basename == "my_project"
+    assert result.project.isdir()
```

Comment:
Should we have a test checking that the package is installable and that the tests pass?
(i.e. pytest, flake8 and mypy?)

Reply:
Done in [357269e](https://github.com/scientificcomputing/generate-python-package/pull/15/commits/357269e2e4e3d5c710ff678ca75bc7f2560a1c94)